src/textual/widgets/_month_calendar.py

```diff
@@ -0,0 +1,107 @@
+from __future__ import annotations
+
+import calendar
+import datetime
+
+from rich.text import Text
+from textual.app import ComposeResult
+from textual.reactive import Reactive, var
+from textual.widget import Widget
+from textual.widgets import DataTable
+
+
+class InvalidMonthNumber(Exception):
+    pass
+
+
+class InvalidWeekdayNumber(Exception):
+    pass
+
+
+class MonthCalendar(Widget):
```

Comment:
Unless I've misunderstood, I think the calendar module is locale aware so already takes care of this?

```
>>> import calendar
>>> list(calendar.day_abbr)
['Mon', 'Tue', 'Wed', 'Thu', 'Fri', 'Sat', 'Sun']
>>> import locale
>>> locale.setlocale(locale.LC_ALL, "de-DE")
'de-DE'
>>> list(calendar.day_abbr)
['Mo', 'Di', 'Mi', 'Do', 'Fr', 'Sa', 'So']
```

Reply:
Ah cool. That's useful. I was thinking more of an override so that we could pass in the strings ourselves, maybe with the locale as a fallback. Not an issue for now though :)